src/AclExtras.php

```diff
@@ -178,10 +178,10 @@ protected function _updateControllers($root, $controllers, $plugin = null)
         // look at each controller
         $controllersNames = [];
         foreach ($controllers as $controller) {
-            $tmp = explode('/', $controller);
+            $tmp = explode(DS, $controller);
             $controllerName = str_replace('Controller.php', '', array_pop($tmp));
             $controllersNames[] = $controllerName;
-            $path = $this->rootNode . '/' . $pluginPath . $controllerName;
+            $path = $this->rootNode . DS . $pluginPath . $controllerName;
```

Comment:
Do we really need to change this too? isnt this the path in the db? Changing that per environment would mean you cannot move the db to another OS without breaking the application